app/src/main/java/org/oppia/app/utility/PackageManagerHelper.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.content.pm.PackageManager
+import javax.inject.Inject
+
+/** Extension functions for PackageManager. */
+@Inject
+lateinit var context: Context
+
+fun PackageManager.getVersionName(): String {
+  return getPackageInfo(context.packageName, /* flags=*/0).versionName
```

Comment:
```suggestion
  return getPackageInfo(context.packageName, /* flags=*/ 0).versionName
```

Feel free to resolve once accepted.